statsd-exporter/test/.gitignore

```diff
@@ -0,0 +1,2 @@
+venv
```

Comment:
The root `.gitignore` contains `venv` which should match this dir and its subdirs. Is there a use case where this is explicitly needed?

Reply:
Maybe from testing this script? https://github.com/astronomer/ap-vendor/blob/master/.circleci/generate_circleci_config.py @sjmiller609 would know

Reply:
What I mean is the presence of `venv` in the root .gitignore should catch all instances of venv in all subdirs.

https://git-scm.com/docs/gitignore

> When deciding whether to ignore a path, Git normally checks gitignore patterns from multiple sources... Patterns read from a .gitignore file in the same directory as the path, or in any parent directory

Reply:
you are right, it's simply unnecessary